addons/website_slides/models/slide_channel.py

```diff
@@ -74,23 +87,26 @@ def _compute_promoted_slide_id(self):
         default=lambda self: self.env['ir.model.data'].xmlid_to_res_id('website_slides.slide_template_shared'))
     visibility = fields.Selection([
         ('public', 'Public'),
-        ('private', 'Private'),
-        ('partial', 'Show channel but restrict presentations')],
+        ('invite', 'Invite')],
         default='public', required=True)
-    group_ids = fields.Many2many(
-        'res.groups', 'rel_channel_groups', 'channel_id', 'group_id',
-        string='Channel Groups', help="Groups allowed to see presentations in this channel")
-    access_error_msg = fields.Html(
-        'Error Message', help="Message to display when not accessible due to access rights",
-        default=lambda s: _("<p>This channel is private and its content is restricted to some users.</p>"),
-        translate=html_translate, sanitize_attributes=False)
+    # members and access
+    partner_ids = fields.Many2many(
+        'res.partner', 'slide_channel_partner', 'channel_id', 'partner_id',
+        string='Members', help="All participants to the channel whatever their state.")
+    member_ids = fields.One2many(
+        comodel_name='res.partner', compute='_compute_member_ids', search='_search_member_ids',
+        string='Confirmed Members')
+    is_member = fields.Boolean(string='Is Member', compute='_compute_member_ids')
+    channel_partner_ids = fields.One2many('slide.channel.partner', 'channel_id', string='Members Information', groups='base.group_system')
+    enroll_msg = fields.Html(
+        'Enroll Message', help="Message to display below the Enroll button to give details on membership details",
```

Comment:
Give details on membership details, that's a lot of details!

Reply:
:male_detective: I found a detail !